fia_api/core/responses.py

```diff
@@ -71,7 +71,9 @@ def from_run(run: Run) -> RunResponse:
         """
         return RunResponse(
             filename=run.filename,
-            experiment_number=run.experiment_number,
+            experiment_number=run.owner.experiment_number
+            if run.owner is not None and run.owner.experiment_number is not None
+            else 0,
```

Comment:
Could experiment numbers of 0 cause problems? E.g. in the frontend will that not make all simple jobs stack together at the start/end in experiment sorting. Is this an issue?

Reply:
So it's either we use 0, or we use None (Null). I believe both will have the same effect. I think None makes the most sense so I will make that change.